lib/vmpooler/pool_manager.rb

```diff
@@ -1,12 +1,17 @@
 module Vmpooler
   class PoolManager
-    def initialize(config, logger, redis, graphite=nil)
+    def initialize(config, logger, redis, graphite=nil, statsd=nil)
       $config = config
 
       # Load logger library
       $logger = logger
 
-      unless graphite.nil?
+      # statsd and graphite are mutex in the context of vmpooler
+      unless statsd.nil?
```

Comment:
Word - I generally just follow what ever pattern was done before, but I'm by no means opposed to cleaning this up as we go :)